frontend/lib/evictionfree/data/faqs-content.tsx

```diff
@@ -228,11 +228,11 @@ export const getEvictionFreeFaqsContent: () => EvictionFreeFaq[] = () => [
       <p>
         <Trans id="evictionfree.deadlineFaq">
           You currently can submit your declaration form at any time between now
-          and May 1, 2021. Once you submit your declaration form via this tool,
-          we will mail and/or email it immediately to your landlord and the
-          courts. If you’re ONLY sending your form via physical mail, send it as
-          soon as possible and keep any proof of mailing and/or return receipts
-          for your records.
+          and August 31, 2021. Once you submit your declaration form via this
+          tool, we will mail and/or email it immediately to your landlord and
+          the courts. If you’re ONLY sending your form via physical mail, send
+          it as soon as possible and keep any proof of mailing and/or return
+          receipts for your records.
         </Trans>
```

Comment:
As we mentioned in our call, you will want to change the `id` here to something like `evictionfree.deadlineFaq2` so that Crowdin actually recognizes this field as having changed!